prow_config.yaml

```diff
@@ -6,3 +6,13 @@ workflows:
     name: e2e
     job_types:
       - presubmit
+  # Run kfctl E2E
+  - app_dir: kubeflow/kubeflow/testing/workflows
+    component: kfctl_go_test
+    name: kfctl-go
+    job_types:
+      - postsubmit
```

Comment:
Can we run this on presubmit? It would be great to prevent breaks from making it in the first place.

Reply:
Yeah, I can do that.
Also, how do I pull in files from a different repo as part of the pre-submit tests?
I'm pull this directory `kubeflow/kubeflow/testing/workflows`, does it have to be cloned in the local before running it?

Reply:
The E2E test (Argo workflow) has a step that checks out all the repositories you need. You can then specify the repositories. Here's where that happens in the current E2E tests.

https://github.com/kubeflow/kubeflow/blob/c7ac19f31893fc0f23730d9d8bfb5bd88dfa2c57/testing/workflows/components/kfctl_go_test.jsonnet#L200

You might need to parameterize that because when the test runs on kubeflow/kubeflow we want to use kubeflow/kubeflow @ PR and kubeflow/manifests at HEAD

but when it runs on kubeflow/manifests we want it to run on
kubeflow/kubeflow @ HEAD and kubeflow/manifests at PR

Reply:
Sorry I missed this comment !